backend/aethel_db/views.py

```diff
@@ -112,6 +114,74 @@ def item_contains_query_string(item: LexicalItem) -> bool:
         return response_object.json_response()
 
 
+class AethelDetailError(Enum):
+    NO_QUERY_INPUT = "NO_QUERY_INPUT"
```

Comment:
None of these errors are actually expected, but it is good to have them, I think.